protocol-specs/generator_outputs/lisk_codec/validArrayEncodings.json

```diff
@@ -0,0 +1,122 @@
+{
+	"title": "Valid array encodings",
+	"summary": "Examples of encoding arrays as required by lisk-codec",
+	"config": {
+		"network": "devnet"
+	},
+	"runner": "lisk_codec",
+	"handler": "validArrayEncodings",
+	"testCases": [
+		{
+			"description": "Encoding of boolean types",
+			"config": {
+				"network": "devnet"
+			},
+			"input": {
+				"arrayInts": {
+					"object": {
+						"list": [
+							3,
+							1,
+							4,
+							1,
+							5,
+							9,
+							2,
+							6,
+							5
+						]
+					},
+					"schema": {
```

Comment:
I think these array schemas are missing the "items" property, indicating the dataType of the array elements.

Reply:
Fixed. Resolving.